t/unit-tests/generate-clar-decls.sh

```diff
@@ -14,6 +14,7 @@ do
 	suite_name=$(basename "$suite")
 	suite_name=${suite_name%.c}
 	suite_name=${suite_name#u-}
+	suite_name=$(echo "$suite_name" | sed 's/-/_/g')
```

Comment:
This is fine as-is, but I think it's more customary in our codebase to use tr(1) instead. E.g. `echo "$suite_name" | tr '-' '_')`.